src/components/standup/index.tsx

```diff
@@ -0,0 +1,139 @@
+        );
+    };
+    useEffect(() => {
+        const isValid = isValidate();
+        setButtonDisable(!isValid);
+    }, [standupUpdate.completed, standupUpdate.planned]);
+
+    const handleFormSubmission = async (
+        event: React.FormEvent<HTMLFormElement>
+    ) => {
+        event.preventDefault();
+        setStandupUpdate({
+            type: 'user',
+            completed: '',
+            planned: '',
+            blockers: '',
+        });
+        await addStandup(standupUpdate);
+    };
+
+    return (
+        <>
+            <section className="container">
+                <div className={styles.standupContainer}>
+                    <div className={styles.standupBanner}>
+                        <p>
+                            You have
+                            <span data-testid="missed-updates">
+                                {totalMissedUpdate} missed
+                            </span>
+                            Standup updates this week
+                        </p>
+                        <p>Let&apos;s try to avoid having zero days </p>
+                    </div>
+                    <div className={styles.standupUpdateContainer}>
+                        <h1>Standup Update</h1>
+                        <form
+                            className={styles.standupForm}
+                            onSubmit={handleFormSubmission}
+                        >
+                            <fieldset className={styles.formFields}>
+                                <label
+                                    className={styles.updateHeading}
+                                    htmlFor="completed"
+                                >
+                                    On {yesterdayDate}
+                                </label>
+                                <FormInputComponent
+                                    dataTestId="yesterday-input-update"
+                                    placeholder="e.g Raised PR for adding new config"
+                                    name="completed"
+                                    value={standupUpdate.completed}
+                                    handleChange={handleChange}
+                                />
+                                <label className={styles.updateHeading}>
+                                    Today
+                                </label>
+                                <FormInputComponent
+                                    dataTestId="today-input-update"
+                                    placeholder="e.g Refactor signup to support Google login"
+                                    name="planned"
+                                    value={standupUpdate.planned}
+                                    handleChange={handleChange}
+                                />
+
+                                <label className={styles.updateHeading}>
+                                    Blockers
+                                </label>
```

Comment:
cant these labels go in the FormInputComponent, You can change the name of component.
Simply calling label again and again makes the doc big.

Please have attributes like htmlFor on the labels.

Reply:
fixed